CHANGELOG.md

```diff
@@ -4,10 +4,46 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v0.5.0](https://github.com/Human-Connection/Human-Connection/compare/v0.4.2...v0.5.0)
-
-> 20 March 2020
-
+#### [v0.6.0](https://github.com/Human-Connection/Human-Connection/compare/v0.4.2...v0.6.0)
```

Comment:
The version intervals look messed up a little

Reply:
I noticed this, and took steps to try and fix it, but was unsuccessful and I didn't know what else to do